core/templates/dev/head/components/CkEditorRteDirective.js

```diff
@@ -158,7 +158,7 @@ oppia.directive('ckEditorRte', [
             } else {
               return '<div type="oppia-noninteractive-' + p3 + '"' +
                      'class="oppia-rte-component-container">' + match +
-                     '<div class="oppia-rte-component-overlay"></div></div>';
```

Comment:
@rakshitkumarcse, can you please explain why do we have to remove this?

Reply:
Hey @DubeySandeep, this was a wrapping around the block components. If you use this, then after clicking the save button and entering in the editing mode again there will be empty line between the paragraph just below it and the block component (say image).

Reply:
@bansalnitish Thanks for answering.

Reply:
Thanks! :)